pages/portfolio.html

```diff
@@ -5,8 +5,38 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="../style/style.css" rel="stylesheet">
 </head>
+
 <body>
+    
+  <nav class="nav">
+    <ul>
+      <li>NAV</li>
+      <li><a href =../pages/index.html> home </a></li>
+      <li><a href =../pages/portfolio.html> - portfolio - </a></li>
+      <li><a href =../pages/about.html> about </a></li>
+    </ul>
+  </nav>
+
+
+  <header class="header">
+    <ul>
+      <li>PORTFOLIO</li>
+      <li>Here they are again, but with a flex-grid.</li>
+      <li><img class="flex-container" src = "../images/top-1701-a.png" alt = "top view of saucer portion of USS Enterprise NCC-1701">
+          <img class="flex-container" src = "../images/top-1701-b.png" alt = "top view of saucer portion of USS Enterprise NCC-1701-B">
+          <img class="flex-container" src = "../images/top-1701-c.png" alt = "top view of saucer portion of USS Enterprise NCC-1701-C">
```

Comment:
These don't actually end up in a flexbox. We would need `display:grid` styling on the `li` element to make this work.